source/include/microapp/cs_MicroappStorage.h

```diff
@@ -99,7 +100,7 @@ class MicroappStorage {
 	void operator=(MicroappStorage const&)  = delete;
 
 	/**
-	 * Keep up whether or not we are currently writing to flash.
+	 * Keep up whether or not we are currently writing to (or erasing) flash.
 	 */
 	bool _writing                           = false;
```

Comment:
There is a function `nrf_fstorage_is_busy` in `nrf_fstorage.h`. Is there a reason we are not using that to query for the state?

Reply:
good one =]